browser/extensions/api/brave_rewards_api.cc

```diff
@@ -270,5 +270,25 @@ void BraveRewardsGetACEnabledFunction::OnGetACEnabled(bool enabled) {
   Respond(OneArgument(std::make_unique<base::Value>(enabled)));
 }
 
+BraveRewardsSaveSettingFunction::~BraveRewardsSaveSettingFunction() {
+}
+
+ExtensionFunction::ResponseAction BraveRewardsSaveSettingFunction::Run() {
+  std::unique_ptr<brave_rewards::SaveSetting::Params> params(
+      brave_rewards::SaveSetting::Params::Create(*args_));
+
+  Profile* profile = Profile::FromBrowserContext(browser_context());
+  RewardsService* rewards_service =
+    RewardsServiceFactory::GetForProfile(profile);
+
+  if (rewards_service) {
+    if (params->key == "enabledMain") {
+      rewards_service->SetRewardsMainEnabled(params->value == "true");
```

Comment:
*nit*
Can we cast here instead of hardcoding string?
(`SetRewardsMainEnabled((bool)params->value);`)

Reply:
Done